e2eTests/Test-AL-Go.ps1

```diff
@@ -1,4 +1,5 @@
-﻿Param(
+﻿[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidGlobalVars', '', Justification = 'Global vars used for local test execution only.')]
```

Comment:
We could, but for GlobalVars - I actually think that is a good general rule to have.
Using Global Vars in tests could also effect other tests - the script scope is much better.
So for these changes - I don't think we should - lets see if there are other Script Analyzer issues which would cause us to do this.